builder/imports/jdk.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2010-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
We're going to need this in more than just the Java CRT (think jruby, or any other JVM language)